From: Peter Lemenkov <lemenkov@gmail.com>
Date: Sat, 25 Mar 2023 20:29:36 +0100
Subject: [PATCH] Bootstrap w/o ygot support

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>

diff --git a/client/gnmi/client.go b/client/gnmi/client.go
index 781b7df..ceb2d2c 100644
--- a/client/gnmi/client.go
+++ b/client/gnmi/client.go
@@ -34,7 +34,6 @@ import (
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc"
 	"google.golang.org/protobuf/proto"
-	"github.com/openconfig/ygot/ygot"
 	"github.com/openconfig/gnmi/client"
 	"github.com/openconfig/gnmi/path"
 	"github.com/openconfig/gnmi/value"
@@ -264,11 +263,8 @@ func subscribe(q client.Query) (*gpb.SubscribeRequest, error) {
 	if q.UpdatesOnly {
 		s.Subscribe.UpdatesOnly = true
 	}
-	for _, qq := range q.Queries {
-		pp, err := ygot.StringToPath(pathToString(qq), ygot.StructuredPath, ygot.StringSlicePath)
-		if err != nil {
-			return nil, fmt.Errorf("invalid query path %q: %v", qq, err)
-		}
+	for _, _ := range q.Queries {
+		pp := &gpb.Path{Origin: "o"}
 		s.Subscribe.Subscription = append(s.Subscribe.Subscription, &gpb.Subscription{Path: pp})
 	}
 	return &gpb.SubscribeRequest{Request: s}, nil
diff --git a/client/gnmi/client_test.go b/client/gnmi/client_test.go
index 8afab8e..d457826 100644
--- a/client/gnmi/client_test.go
+++ b/client/gnmi/client_test.go
@@ -24,7 +24,6 @@ import (
 
 	"github.com/kylelemons/godebug/pretty"
 	"google.golang.org/grpc"
-	"github.com/openconfig/ygot/ygot"
 	"github.com/openconfig/gnmi/client"
 	"github.com/openconfig/gnmi/testing/fake/gnmi"
 	"github.com/openconfig/gnmi/testing/fake/testing/grpc/config"
@@ -417,10 +416,7 @@ func TestGNMIWithSubscribeRequest(t *testing.T) {
 }
 
 func stringToPath(p string) *gpb.Path {
-	pp, err := ygot.StringToPath(p, ygot.StructuredPath, ygot.StringSlicePath)
-	if err != nil {
-		panic(err)
-	}
+	pp, := &gpb.Path{Origin: "o"}
 	return pp
 }
 
